packages/react-server/src/ReactFlightServerConfigStream.js

```diff
@@ -156,6 +151,17 @@ export function processImportChunk(
   return stringToChunk(row);
 }
 
+export function processDirectiveChunk(
+  request: Request,
+  id: number,
+  payload: ReactClientValue,
+): Chunk {
+  // $FlowFixMe[incompatible-type] stringify can return null
+  const json: string = stringify(payload, request.toJSON);
```

Comment:
Same thinking here: `request.toJSON` could be omitted, no?

Reply:
Isn't this still superfluous, even after your latest changes? Or am I missing something?

Reply:
yup good call

Reply:
someday conceivably we could have directives that encode references to other model values and this may be necessary but we don't today and it would require work on the client to opt into that too